### PR TITLE
[COREUS-217] add USA devs

### PR DESCRIPTION
**Pull request by aj-65**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,9 @@ free to work in different squads if the work load justifies it.
 | Tomas Navickas                | Healthcheck              | [GitHub](https://github.com/iTomkinas) |
 | Bogdan-Andrei Fotescu         | Monitor                  | [GitHub](https://github.com/rakatan) |
 | Luca Nicoletti                | Monitor                  | [GitHub](https://github.com/lnicolet) <br> [Medium](https://medium.com/@luca.nicolett) <br> [Twitter](https://twitter.com/luca_nicolett) |
+| Antonio Johnson                | USA                  | [GitHub](https://github.com/aj-65) |
+| Sean Gallagher                 | USA                  | [GitHub](https://github.com/seangallagherbabylon) |
+| Jayant Jere                    | USA                  | [GitHub](https://github.com/j-android) |
 
 We are now split into 2 offices, one in South Kensington (HQ) and
```
Comment (jcmsalves) [inline README.md]: Can you also update this section and mention we have the Austin office as well? And in the job descriptions docs add the 🇺🇸 as well 🙂

**Review by jcmsalves — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (aj-65) [reply README.md]: Done. I don't have permission to push new branches so I had to create patches in github. I couldn't figure out how to do all the changes in 1 branch. 

So, all 3 pull requests are mine: https://github.com/Babylonpartners/android-playbook/pulls

Comment (jcmsalves) [reply README.md]: That's weird, I'll check the permissions and let you know.

**Review by jcmsalves — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jcmsalves) [reply README.md]: For some reason, Jayant and Sean were already on the `Android` team in GitHub but you were not. Just added you as well so now you can create and push branches normally.

Comment (aj-65) [reply README.md]: Thank you!

**Pull request merged**